bigquery_etl/cli/utils.py

```diff
@@ -25,10 +25,14 @@ def is_valid_file(ctx, param, value):
     return value
 
 
-def is_authenticated(project_id="moz-fx-data-shared-prod"):
+def is_authenticated(project_id=None):
     """Check if the user is authenticated to GCP and can access the project."""
     client = bigquery.Client()
```

Comment:
This will raise an exception [if the default credentials cannot be acquired](https://googleapis.dev/python/bigquery/latest/generated/google.cloud.bigquery.client.Client.html#google.cloud.bigquery.client.Client)